src/webgl/p5.Shader.js

```diff
@@ -277,12 +272,11 @@ p5.Shader.prototype.useProgram = function() {
  */
 p5.Shader.prototype.setUniform = function(uniformName, data) {
   //@todo update all current gl.uniformXX calls
-
-  var uniform = this.uniforms[uniformName];
-  if (!uniform) {
-    //@todo warning?
+  if (!(uniformName in this.uniforms)) {
     return;
   }
+
+  var uniform = this.uniforms[uniformName];
```

Comment:
I don't think it's necessary to do the dictionary lookup twice here. the original code had the same effect and was more performant.